src/stores/redis.rs

```diff
@@ -341,6 +341,15 @@ where
         }
     }
 
+    fn cache_clear(&self) -> Result<(), Self::Error> {
+        let mut conn = self.pool.get()?;
+
+        // https://redis.io/commands/flushdb/
+        let _: () = redis::cmd("FLUSHDB").query(&mut *conn)?;
```

Comment:
Wouldn't this delete someone's entire redis db? I think a `scan` + `delete` would need to be done instead, using the `self.namespace` + `self.prefix` in the `scan`

Reply:
Oh I just assumed the Redis DB was already unique per cache. I can fix this.

Reply:
Ty!

Reply:
I took a shot at it but needing two connections seems weird. `scan_match` needs `&mut self`, though, so I can't _do_ anything with the deserialized values without opening another connection or collecting them all into memory first.

https://docs.rs/redis/latest/redis/trait.Commands.html#method.scan_match